FreeRTOS/FreeRTOS
# Fix MQTT keep alive and winsim logging warnings

Description
-----------
This PR fixes compilation warnings for the MQTT winsim keep alive demo and the winsim logging module in Visual Studio 2022.

Test Steps
-----------
Build on Visual Studio 2022.

Checklist:
----------



Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
